places/tests.py

```diff
@@ -0,0 +1,89 @@
+from django.test import TestCase
+from . import mock
+import responses, requests, os
+
+API_KEY = os.getenv('API_KEY')
+Q = 'lagos'
+SOURCE = 'google'
+LNG = 3.3792057
+LAT = 6.5243793
+PLACE_ID = "ChIJwYCC5iqLOxARy9nDZ6OHntw"
+COMMON = "https://maps.googleapis.com/maps/api/place/"
+URL = f"{COMMON}findplacefromtext/json?inputtype=textquery&key={API_KEY}&input="
+ZERO_SEARCH_URL = f"{URL}xxxx"
+SEARCH_URL = f"{URL}{Q}"
+SEARCH_URL_WITH_LOCATION = f"{SEARCH_URL}&locationbias=point:{LNG},{LAT}"
+DETAILS_URL = f"{COMMON}details/json?place_id={PLACE_ID}&key={API_KEY}"
+DETAILS = mock.details
```

Comment:
be explicit about what you are importing
do 
`from .mock import no_candidate, candidate... ` instead